src/test/pybind/test_rbd.py

```diff
@@ -779,7 +790,7 @@ def test_copy3(self):
         self._test_copy(features, self.image.stat()['order'],
                         self.image.stripe_unit(), self.image.stripe_count())
 
-    @attr('SKIP_IF_CRIMSON')
+    @pytest.mark.skip_if_crimson
```

Comment:
I see the following warning suggesting to register the mark:
```
/usr/lib/python3/dist-packages/_pytest/mark/structures.py:331
  /usr/lib/python3/dist-packages/_pytest/mark/structures.py:331: PytestUnknownMarkWarning: Unknown pytest.mark.skip_if_crimson - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/latest/mark.html
    warnings.warn(

-- Docs: https://docs.pytest.org/en/latest/warnings.html
```